test/dialect/postgresql/test_reflection.py

```diff
@@ -907,6 +907,74 @@ def go():
             ],
         )
 
+    @testing.fails_if("postgresql < 8.3", "index ordering not supported")
+    @testing.provide_metadata
+    def test_index_reflection_with_sorting(self):
+        """reflect indexes with sorting options set"""
+        metadata = self.metadata
+
+        t1 = Table(
+            "party",
+            metadata,
+            Column("id", String(10), nullable=False),
+            Column("name", String(20)),
+            Column("aname", String(20)),
+        )
+        metadata.create_all()
+
+        # check ASC, DESC options alone
+        testing.db.execute("""
+            create index idx1 on party 
+                (id, name ASC, aname DESC)
+        """)
+
+        # check DESC w/ NULLS options
+        testing.db.execute("""
+          create index idx2 on party 
+                (name DESC NULLS FIRST, aname DESC NULLS LAST)  
+        """)
+
+        # check ASC w/ NULLS options
+        testing.db.execute("""
+          create index idx3 on party 
+                (name ASC NULLS FIRST, aname ASC NULLS LAST)  
+        """)
+
+        # reflect data
+        m2 = MetaData(testing.db)
+        t2 = Table("party", m2, autoload=True)
+        assert len(t2.indexes) == 3
```

Comment:
another thing that is newer and likely not everywhere in this old test suite:

`    eq_(len(t2.indexes), 3)
`

since then, py.test detects and instruments `==` signs, unittest added assertEquals(), bah...we're just trying to stick w/ the one :)